src/styles/Card.scss

```
@@ -0,0 +1,111 @@
+@import './constants.scss';
+
+
+$imageDimensionDesktop:300px;
+$imageDimensionMobile:425px;
+$cardMargin:20px;
+
+.card-object {
+    margin:$cardMargin;
+    // width:$desktopContentWidth;
+    width:100%;
+    margin-left: auto;
+    margin-right: auto;
+    box-shadow: 30px 10px $primaryDark;
+}
+
+.card-is-in-timeline-true.card-justification-left {
+    width:$timelineCardWidth;
+    margin-right: 60vw;
+}
+
+.card-is-in-timeline-true.card-justification-right {
+    width:$timelineCardWidth;
+    margin-left: 60vw;
+}
+
+.card-link {
+    background-color: $secondaryLight;
+    width:100%;
+    display: flex;
+    flex-direction: row;    
+    padding:10px;
+}
+
+.card-image-on-right {
+    flex-direction: row-reverse;    
+}
+
+.card-image {
+    object-fit: cover;
+    height: $imageDimensionDesktop;
+    width: $imageDimensionDesktop;
+    float:right;
+}
+
+.card-text-block{
+    display: flex;
+    flex-direction: column;
+    align-items: left;
+    justify-content: center;
+}
+
+.card-title {
+    background-color: $primaryColor;
+    font-size: $titleFontSize;
+    color: $primaryText;
+    margin-block: 0px;
+    padding-left:10px;
+    padding-bottom: 10px;
+}
+
+.card-body {
+    background-color: $primaryLight;
+    font-size: $bodyFontSize;
+    color: $secondaryText;
+    margin-block: 0px;
+    margin-left:$cardMargin;
+    margin-right:$cardMargin;
+    padding-left:10px;
+    padding-bottom: 15px;
+}
+
+.timeline-card-holder{
+    width:70vw;
+    margin-left:auto;
+    margin-right:auto;
+}
+
+@media (max-width: 430px) {
+
+    .card-object {
+        width:100vw;
+        box-shadow: 0px 0px;
+        margin-left: auto;
+        margin-right: auto;
+    }
+
+    .card-is-in-timeline-true.card-justification-left {
+        margin-right: auto;
+        margin-left: auto;
+    }
+    
+    .card-is-in-timeline-true.card-justification-right {
+        margin-right: auto;
+        margin-left: auto;
+    }
```

```suggestion
    .card-is-in-timeline-true {
        .card-justification-left {
            margin-right: auto;
            margin-left: auto;
        }
        
        .card-justification-right {
            margin-right: auto;
            margin-left: auto;
        }
    }
```

As well as this I would suggest changing the name of card is in timeline to maybe `timeline-card` and then not have the class if it is false

---

I would but i think time is an issues so would prefer to do this down the line. Lets discus.